### PR TITLE
Update website_robots.xml

### PR DESCRIPTION
**Pull request by benzir123**
Fix robots. txt file.

### CODE REVIEW

File: addons/website/wizard/website_robots.xml [hunk 1]
```diff
@@ -13,8 +13,11 @@
                     </a>
                     <br/><br/>
                     Example of rule:<br/>
-                    <code t-translation="off" class='ms-4'>Disallow: /web/login</code><br/>
-                    <code t-translation="off" class='ms-4'>Allow: *</code>
+                    
+                   <code t-translation="off" class='ms-4'>User-agent: *</code><br/>
+                       <code t-translation="off" class='ms-4'>Disallow:</code><br/>
+                           <code t-translation="off" class='ms-4'>Disallow: /web/login</code><br/>
+                                <code t-translation="off" class='ms-4'>Sitemap: https://www.quality-galore.com/sitemap.xml</code>
```
Comment (rdeodoo) [inline addons/website/wizard/website_robots.xml:20]: `https://www.quality-galore.com` ?

**Review by rdeodoo — CHANGES_REQUESTED** [review]
Hello,

Thanks for your contribution but we won't be able to go further with the current state of this PR.

> Update website_robots.xml
> Fix robots. txt file.

Is not enough (and not correctly formatted) for us to understand what is trying to be achieved here.

Could you please elaborate a bit more?
Note that the sitemap doesn't have to be added in the robots.txt (it can but it's not mandatory and doesn't really change anything if you have a sitemap.xml existing AFAIK)

File: addons/website/wizard/website_robots.xml (hunk 1 above)
Comment (benzir123) [reply addons/website/wizard/website_robots.xml:20]: yes

Comment (rdeodoo) [reply addons/website/wizard/website_robots.xml:20]: no

**Pull request closed without merging**